Solutions/Corvus.Leasing.Azure/Corvus/Leasing/AzureLeaseProviderOptions.cs

```diff
@@ -14,6 +14,6 @@ public class AzureLeaseProviderOptions
         /// <summary>
         /// Gets or sets the storage account connection string.
         /// </summary>
-        public string StorageAccountConnectionString { get; set; }
+        public string? StorageAccountConnectionString { get; set; }
```

Comment:
Isn't it an error for this ever to be non-null?

Or is this in anticipation of possibly adding other approaches such as key vault?